srcpkgs/gnupg-smartcard-udev-rules/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gnupg-smartcard-udev-rules'
+pkgname=gnupg-smartcard-udev-rules
+version=0.0.20220713
+revision=1
+_commit=6cb4406c7d6962e793da0dc84ab78f186cb8fa6e
+build_style=fetch
+short_desc="GnuPG udev rules for use with OpenPGP smartcards"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://salsa.debian.org/debian/gnupg2/-/blob/debian/unstable/debian/scdaemon.udev"
+distfiles="https://salsa.debian.org/debian/gnupg2/-/raw/${_commit}/debian/scdaemon.udev"
+checksum=23e36cb8381df9461b73abdab838dbc927a475fb22b14dd53b7f8192ccdd6023
+
+do_install() {
+	sed -r 's/(, ENV\{.+\}=\".+\")+$/, GROUP="plugdev", TAG+="uaccess"/gm' -i scdaemon.udev
```

Comment:
I'd like to use vsed here, but doing so means I can't use a regex here so I'd have to write

```
, ENV{ID_SMARTCARD_READER}="1", ENV{ID_SMARTCARD_READER_DRIVER}="gnupg"
```
instead of 
```
(, ENV\{.+\}=\".+\")+$
```